include/seqan/sequence/adapt_array_pointer.h

```diff
@@ -102,54 +102,30 @@ struct DefaultOverflowExplicit< TValue const [SIZE] >
 };
 
 template <typename TValue>
-struct IsContiguous;
-
-template <typename TValue>
-struct IsContiguous< TValue * >
-{
-    typedef True Type;
-    enum { VALUE = true };
-};
+struct IsContiguous< TValue * > : public True {};
 
 template <typename TValue, size_t SIZE>
-struct IsContiguous< TValue [SIZE] >
-{
-    typedef True Type;
-    enum { VALUE = true };
-};
+struct IsContiguous< TValue [SIZE] > : public True {};
 
 template <typename TValue, size_t SIZE>
-struct IsContiguous< TValue const [SIZE] >
-{
-    typedef True Type;
-    enum { VALUE = true };
-};
+struct IsContiguous< TValue const [SIZE] > : public True {};
 
 template <typename TValue>
-struct IsSequence< TValue * >
-{
-    typedef True Type;
-    enum { VALUE = true };
-};
+struct IsSequence< TValue * > : public True {};
+
 template <typename TValue, size_t SIZE>
-struct IsSequence< TValue [SIZE] >
-{
-    typedef True Type;
-    enum { VALUE = true };
-};
+struct IsSequence< TValue [SIZE] > : public True {};
+
 template <typename TValue, size_t SIZE>
-struct IsSequence< TValue const [SIZE] >
-{
-    typedef True Type;
-    enum { VALUE = true };
-};
+struct IsSequence< TValue const [SIZE] > : public True {};
 
 // ----------------------------------------------------------------------------
 // Concept Sequence
 // ----------------------------------------------------------------------------
 
-template <typename TValue>
-SEQAN_CONCEPT_IMPL((TValue *), (ContainerConcept));
+// NOTE(h-2): why is this broken?
+// template <typename TValue>
+// SEQAN_CONCEPT_IMPL((TValue *), (ContainerConcept));
 
```

Comment:
If the tests pass this is the last issue. As far as I am concerned I would just leave it like this, because making a regular pointer a container:
  * makes no sense
  * is not used anywhere
  * is not documented so no one expects it